index.html

```diff
@@ -195,6 +192,140 @@ <h2>ts-note</h2>
+					<h4>Subtitle</h4>
+					<ts-list-item selectable title="Simplest" subtitle="subtitle text goes here"></ts-list-item>
+					<ts-list-item
+						selectable
+						icon="ada"
+						title="With icon on the left"
+						subtitle="subtitle text goes here"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						icon="ada"
+						icon-right="remove"
+						title="with the icon on both side"
+						subtitle="subtitle text goes here"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						icon-right="remove"
+						title="with the icon on the right"
+						subtitle="subtitle text goes here"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						selected
+						icon-right="remove"
+						title="with the icon on the right"
+						subtitle="subtitle text goes here"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						disabled
+						icon-right="remove"
+						title="with the icon on the right"
+						subtitle="subtitle text goes here"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						selected
+						disabled
+						icon-right="remove"
+						title="with the icon on the right"
+						subtitle="subtitle text goes here"
+					></ts-list-item>
+					<hr />
+					<h4>Edge cases</h4>
+					<ts-list-item
+						selectable
+						icon="ada"
+						icon-right="remove"
+						title="some extraaaaaa long text title to see what will happen to them in the list item"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						icon="ada"
+						icon-right="remove"
+						subtitle="with subtitle"
+						title="some extraaaaaa long text title to see what will happen to them in the list item"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						selected
+						disabled
+						icon="ada"
+						icon-right="remove"
+						subtitle="Lorem ipsum dolor sit amet, consectetur adipisicing elit. A aspernatur at autem doloribus ducimus enim eum ex fuga, necessitatibus nulla quas quasi qui quisquam, rerum sapiente tempora tenetur vel, voluptas."
+						title="some extraaaaaa long text title to see what will happen to them in the list item"
+					></ts-list-item>
+					<ts-list-item
+						selectable
+						selected
+						disabled
+						dir="rtl"
+						icon="ada"
+						icon-right="remove"
+						subtitle="Lorem ipsum dolor sit amet, consectetur adipisicing elit. A aspernatur at autem doloribus ducimus enim eum ex fuga, necessitatibus nulla quas quasi qui quisquam, rerum sapiente tempora tenetur vel, voluptas."
+						title="some extraaaaaa long text title to see what will happen to them in the list item"
+					></ts-list-item>
+					<hr />
+					<h4>RTL</h4>
+					<ts-list-item
+						selectable
+						dir="rtl"
+						icon="ada"
+						icon-right="remove"
+						subtitle="Lorem ipsum dolor sit amet, consectetur adipisicing elit. A aspernatur at autem doloribus ducimus enim eum ex fuga, necessitatibus nulla quas quasi qui quisquam, rerum sapiente tempora tenetur vel, voluptas."
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 224 characters long line (which is greater than 160 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AMaxLineLengthCheck)